examples/tensorflow/image_recognition/keras_models/inception_resnet_v2/quantization/ptq/run_benchmark.sh

```diff
@@ -13,17 +13,14 @@ function init_params {
   for var in "$@"
   do
     case $var in
-      --config=*)
-          config=$(echo $var |cut -f2 -d=)
-      ;;
       --input_model=*)
           input_model=$(echo $var |cut -f2 -d=)
       ;;
       --mode=*)
           mode=$(echo $var |cut -f2 -d=)    
       ;;
-      --eval_data=*)
-          eval_data=$(echo $var |cut -f2 -d=)    
+      --dataset_location=*)
+          dataset_location=$(echo $var |cut -f2 -d=)    
       ;;
```

Comment:
![image](https://user-images.githubusercontent.com/51692656/205591139-90ac3182-31c6-45d5-85b4-d888ddaf7f90.png)
Please set 4 parameters for run_benchmark.sh